runtime/autoload/man.vim

```diff
@@ -118,15 +118,12 @@ endfunction
 " Expects a string like 'access' or 'access(2)'.
 function s:parse_page_and_section(sect, str) abort
   try
-    let save_isk = &iskeyword
-    setlocal iskeyword-=(,)
-    let page = substitute(a:str, '(*\(\k\+\).*', '\1', '')
-    let sect = substitute(a:str, '\(\k\+\)(\([^()]*\)).*', '\2', '')
+    let page = substitute(a:str, '\([-.[:alnum:]_]\+\).*', '\1', '')
+    let sect = substitute(a:str, '\([-.[:alnum:]_]\+\)(\([^()]*\)).*', '\2', '')
```

Comment:
I suggested such change earlier, but got confused by `(*` absent in the second regex. If these regular expressions are really fine, `matchlist()` could safely be used.